RecoPixelVertexing/PixelTriplets/plugins/GPUCACell.h

```diff
@@ -84,7 +84,7 @@ class GPUCACell {
 
     auto r1 = otherCell.get_inner_r(hh);
     auto z1 = otherCell.get_inner_z(hh);
-    bool aligned = areAlignedRZ(r1, z1, ri, zi, ro, zo, ptmin, thetaCut);
+    bool aligned = areAlignedRZ(r1, z1, ri, zi, ro, zo, ptmin, 2*thetaCut);
```

Comment:
I prefer to keep it as is for the time being NOT to change the behavior on CPU
All cuts need to be optimized...